### PR TITLE
DcMotor Constructor for Motor Wrappers

### PR DESCRIPTION
**Pull request by JIceberg**
# DcMotor Constructor for Motor Wrappers

## What kind of change does this PR introduce?
* Feature

## Did this PR introduce a breaking change?
_A breaking change includes anything that breaks backwards compatibility either at compile or run time._
* No

__Please make sure your PR satisfies the requirements of [the contributing page](CONTRIBUTING.md)__

### CODE REVIEW

**Comment by JIceberg** [comment]
I want to also add some unit testing now for this, and see if I can get that working. This might be a reach goal. If not done within the next month, will abandon for later.

File: core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java [hunk 1]
```diff
@@ -238,6 +238,32 @@ public DcMotor.ZeroPowerBehavior getBehavior() {
     public Motor() {
```
Comment (lorenzogood) [inline core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java:238]: Just out of curiosity, why is there an empty default constructor.

**Review by lorenzogood — APPROVED** [review]

File: core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java (hunk 1 above)
Comment (JIceberg) [reply core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java:238]: For OOP shenanigans

**Review by Powercube7 — APPROVED** [review]
LGTM!

**Pull request merged**